src/docfx/config/ConfigLoader.cs

```diff
@@ -86,6 +88,15 @@ where glob(configPath)
             var (configErrors, config) = JsonUtility.ToObject<Config>(configObject);
             errors.AddRange(configErrors);
 
+            // Track docfx config telemetry
+            Telemetry.TrackEvent(
+                docfxConfigFileName,
+                new Dictionary<string, string>
+                {
+                    ["Config"] = docfxConfigFileContent,
```

Comment:
`docfxConfigFileContent` is YAML for `docfx.yml`, do you want to always produce json?
```suggestion
                    ["Config"] = docfxConfig.ToString(),
```

Reply:
I think it is okay to track different format for different input source - plus the event name is different if the config file is actually `docfx.yml`